packages/data/addon/serializers/facts/interface.ts

```diff
@@ -9,8 +9,8 @@ import { RequestV1, RequestV2 } from 'navi-data/adapters/facts/interface';
 type ResponsePayload = any;
 
 export interface ResponseV1 {
-  rows: Array<object>;
-  meta: {
+  rows: Array<Record<string, unknown>>;
+  meta?: {
```

Comment:
Are we sure there aren't places (non-ts) that are expecting an object?

Reply:
👍 , it looks like in some tests we just don't set it so I added `|| {}` as a fallback